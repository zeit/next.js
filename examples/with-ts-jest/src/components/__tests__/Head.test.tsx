import React from "react";
import { act, create, ReactTestRenderer } from "react-test-renderer";
import Header from "../Head";

describe("COMPONENT Header", () => {
  test("WILL RENDER", () => {
    let root: ReactTestRenderer;

    act(() => {
      root = create(<Header />);
    });

    expect(root.toJSON()).toMatchSnapshot();
  });
});
