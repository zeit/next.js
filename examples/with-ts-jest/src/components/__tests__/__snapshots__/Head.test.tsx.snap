// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT Header WILL RENDER 1`] = `null`;
